clients/client-python/gravitino_client/core/gravitino_client.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+import requests
+from typing import Dict
+
+class GravitinoClient:
+    """
+    Gravitino Client for interacting with the Gravitino API, allowing the client to list, load,
+    create, and alter Metalakes.
+
+    Attributes:
+        base_url (str): The base URL of the Gravitino API to which the client will make requests.
+
+    Args:
+        base_url (str): The base URL for the Gravitino API.
+    """
+
+    def __init__(self, base_url):
+        """
+        Initializes a new instance of the GravitinoClient.
+
+        Args:
+            base_url (str): The base URL for the Gravitino API where the client will send requests.
+        """
+        self.base_url = base_url
+
+    def getVersion(self) -> Dict[str, str]:
```

Comment:
I found the Java DTO is declared on the `common` module. Do we have a plan to support multi-language clients? If so, we may face many repetitive development tasks. A possible choice is to use a tool like [OpenAPI Generator](https://openapi-generator.tech/) to generate the code automatically. In this way, we can better ensure the consistency of interfaces between different clients. Maybe we can create a new issue to track this. In Python client, we can implement it in the current way to avoid affecting the progress of this work. What do you think? @jerryshao @xunliu @justinmclean @SophieTech88

Reply:
I think if the open api can generate object models like Java DTOs, then it is a way to try, maybe we can spend time on investigating it before writing codes.

Reply:
@Lanznx In https://github.com/datastrato/gravitino/pull/2342, I'm trying to use the OpenAPI generator to generate the DTO. you can take a look, and have a try.

Reply:
The `VersionDTO` is not included in the `open-api.yaml`, maybe we need to solve it first.